### PR TITLE
Jenkinsfile and Pod Label

### PR DESCRIPTION
**Pull request by bgeesaman**
Jenkinsfile podtemplate, formatting, pod label

### CODE REVIEW

File: container/Dockerfile [hunk 1]
```diff
@@ -14,4 +14,4 @@
 
 FROM nginx:alpine
 
-RUN apk add --no-cache curl=7.64.0-r2
+RUN apk add --no-cache curl
```
Comment (chrislovecnm) [inline container/Dockerfile]: That will not pass our dockerfile linter ...

Comment (chrislovecnm) [reply container/Dockerfile]: Well it should not have passed our limiter :( . Can we use the gcloud container and remove this one??

Comment (bgeesaman) [reply container/Dockerfile]: The validate script hits the app as nginx and also execs in and runs curl, so leaving this as-is is probably best.  To strike a balance between linting correctness and having to determine the exact version and retest the entire demo every time alpine releases a new version of curl, I am using a fuzzy version match.

**Review by chrislovecnm — APPROVED** [review]
/lgtm

**Pull request merged**